lib/event.go

```diff
@@ -28,6 +28,26 @@ const (
 	EventStatusClosed           EventStatus = "CLOSED"
 )
 
+// Type for the rejection.
+const(
+	PRE RejectionType = "Pre"
+	POST RejectionType = "Post"
+)
```

Comment:
Previously the Design contained a rejection type, currently we don't have this info from the API, but i kept it, maybe it will be needed.